features/edit_category.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Edit Category
+  In order to maintain typo blog
+  As an admin
+  I want to edit existing categories
+
+  Background:
+    Given the blog is set up
+    And I am logged into the admin panel
+
+  Scenario: Successfully Edit Categories
+    Given I have categories named Fluffy, Shiny
```

Comment:
It seems like this scenario with two different categories is adding extra steps. Since you are testing the edit functionality, i would start with only one category like Fluffy, and remove the extra steps related to Shiny.